http/src/main/java/io/micronaut/http/HttpHeaders.java

```diff
@@ -499,6 +499,103 @@ public interface HttpHeaders extends Headers {
      */
     String X_AUTH_TOKEN = "X-Auth-Token";
 
+    List<String> STANDARD_HEADERS = Arrays.asList(
```

Comment:
should be immutable

Reply:
and should have javadoc

Reply:
https://github.com/micronaut-projects/micronaut-core/pull/8903/commits/998f634d13f9d15a91e9cab4d5e233cfc1525b27 and https://github.com/micronaut-projects/micronaut-core/pull/8903/commits/ec236c48dc22339a9b60a62930b01ffa534de15d